### PR TITLE
cmd/atlascmd: add supported format for dbmate

### PR DESCRIPTION
**Pull request by iwata**
## What

SSIA

## Why

To integrate with [amacneil/dbmate](https://github.com/amacneil/dbmate) easily.

### CODE REVIEW

File: cmd/atlascmd/migrate.go [hunk 1]
```diff
@@ -430,6 +430,7 @@ const (
 	formatGoose         = "goose"
 	formatFlyway        = "flyway"
 	formatLiquibase     = "liquibase"
+	formatDbmate        = "dbmate"
```
Comment (a8m) [inline cmd/atlascmd/migrate.go:433]:
```suggestion
	formatDBMate        = "dbmate"
```
Although the project uses `Dbmate`, I prefer to follow the [Go initialisms rule](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). @masseelch, your thoughts?

**Review by a8m — APPROVED** [review]
💯

File: cmd/atlascmd/migrate.go (hunk 1 above)
Comment (masseelch) [reply cmd/atlascmd/migrate.go:433]: Agree to that. Reviewing now.

**Review by masseelch — APPROVED** [review]
:rocket: Thanks @iwata for this contribution :blush:

File: cmd/atlascmd/migrate.go (hunk 1 above)
Comment (iwata) [reply cmd/atlascmd/migrate.go:433]: But the author of `dbmate` seems to describe it as `Dbmate`, so I believe Dbmate is an intrinsic nouns.
https://github.com/amacneil/dbmate#dbmate

Comment (a8m) [reply cmd/atlascmd/migrate.go:433]: I know. The project name is `Dbmate`, but I prefer the variable name to be `DBMate`. Sorry for being annoying 😉 

Would you mind changing it to `DBMate`? Thanks

File: sql/sqltool/tool.go [hunk 2]
```diff
@@ -46,6 +46,14 @@ var (
 {{ $change.Cmd }};
 {{ with $change.Reverse }}--rollback: {{ . }};{{ end }}
 {{ end }}`,
+	)
+	// DbmateFormatter returns a migrate.Formatter compatible with amacneil/dbmate.
+	DbmateFormatter = templateFormatter(
```
Comment (a8m) [inline sql/sqltool/tool.go:51]: Please, rename this one as well 🙏

**Comment by a8m** [comment]
@iwata, any reason for closing this?

**Comment by a8m** [comment]
I accept your argument and agree with it. You're welcome to contribute it in case it's still relevant. Thanks

**Pull request closed without merging**